scripts/generator.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+v='\xE2\x9c\x94'
```

Comment:
@jonathanadler 	well if you're going to the lengths of this, then i wouldn't advise bash script. i thought you wanted just a yes/no prompt

Reply:
Well, it's already done 🤷‍♂ 
I prefer bash because with node the only option is to spawn children which I don't like to handle

Reply:
@argshook so what do you say?

Reply:
this code is a little scary and could bite back in future but im not against it :)

i would probably not add any colors or the loop, just a simple `y/n` and that's it. But you're the maintainer, it's your homeland, you can do everything, the only limit is yourself (https://html5zombo.com/)

Reply:
😂